src/driver/amdxdna/amdxdna_mailbox.c

```diff
@@ -675,7 +713,14 @@ void xdna_mailbox_stop_channel(struct mailbox_channel *mb_chann)
 		return;
 
 	/* Disalbe an irq and wait. This might sleep. */
+#ifdef AMDXDNA_DEVEL
+	if (mailbox_polling)
+		del_timer_sync(&mb_chann->event_timer);
```

Comment:
Sorry, I was giving a wrong pointer. The linux said "Do not use in new code. Use timer_delete_sync() instead."

```suggestion
		timer_delete_sync(&mb_chann->event_timer);
```